templates-java/src/test/java/br/com/templates_java/config/jms_aws_sqs/ProducerMessageGeneratorTest.java

```diff
@@ -0,0 +1,65 @@
+package br.com.templates_java.config.jms_aws_sqs;
+
+import br.com.generator.core.GeneratorOptions;
+import br.com.templates_java.helper.LoadTemplateTester;
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+public class ProducerMessageGeneratorTest {
+
+    private LoadTemplateTester loadTemplateTester;
```

Comment:
Consider making `loadTemplateTester` `final` since it's not reassigned after initialization